src/main/java/onboarding/Problem4.java

```diff
@@ -2,7 +2,25 @@
 
 public class Problem4 {
     public static String solution(String word) {
-        String answer = "";
+        StringBuilder sb = new StringBuilder();
+        for(int i = 0; i < word.length(); i++) {
+            sb.append(convertInReverse(word.charAt(i)));
+        }
+        String answer = sb.toString();
         return answer;
```

Comment:
```suggestion
        return sb.toString();
```
`answer` 변수는 굳이 없어도 될 것 같습니다.